README.md

```diff
@@ -45,15 +33,47 @@ COMPOSE_DOCKER_CLI_BUILD=1 docker-compose build
 docker-compose up
 ```
 
-The server application will be available on the port you configured with `DOCKER_LDES_SERVER_PORT` (or port 8080 by default).
-
-#### IntelliJ
+### Maven
 
-To start an LDES Server locally in your IntelliJ IDE, you'll need to add components for storage, ingestion and publication to your classpath.
-In IntelliJ you can do this by updating your project structure.
-Go to the `ldes-server-application` module and add jars for storage and publication.
-Your project structure will look as follows: ![ClasspathDependencies](image/ClasspathDependencies.png)
+To locally run the LDES Server in maven, the following Maven command can be executed:
 
-Now you only need to provide the correct environment variables by overriding the placeholders in [application.yml](ldes-server-application/src/main/resources/application.yml).
+```mvn
+mvn spring-boot:run
+```
 
-And you can spin up your own LDES Server.
\ No newline at end of file
+This will run a clean slate LDES Server depending on your needs, one or of the following maven profiles can be activated:
+
```

Comment:
Can you mention were people can find this application config => so the pom.xml. Think like not everyone has java / maven knowledge :-)

Reply:
This could be added, but can we not assume that anyone who stumbles upon a maven/java/spring project will either have this knowledge or first get familiar with the technology?

Reply:
I see what you mean... maybe create a ref link `application config` which is mapped to the pom.xml  => is min effort :-)